animated-transformer/src/lib/transformer/transformer_gtensor.ts

```diff
@@ -316,6 +316,12 @@ function gelu(x: tf.Tensor) {
   return tf.mul(x, cdf);
 }
 
+export function computeMaskedAffinities(
```

Comment:
Skip introducing a new function to just wrap a single other call; you can remove 'computeMaskedAffinities' and just use `qk.triangularMask('keyPos', 'queryPos', -Infinity);` in it's place.